template.tpl

```diff
@@ -361,7 +413,7 @@ ___WEB_PERMISSIONS___
                   },
                   {
                     "type": 8,
-                    "boolean": false
+                    "boolean": true
```

Comment:
@akersten  Why are we changing these? I think they should have the options of true and false...

Reply:
To support the built-in GTM unit testing, the template needs permission to read the consent state. These flags correspond to the permissions on this tab where I had to update these to `true` for the `Read` permission:

<img width="1138" alt="image" src="https://github.com/truevault/gtm-template-polaris-cmp/assets/2402493/9e617efc-38a7-497e-bdcf-a898e0eb312e">

Reply:
Ooh that makes sense, thanks for clarifiying!